meadow-client/src/Blockly/Generator.purs

```diff
@@ -0,0 +1,119 @@
+module Blockly.Generator where
+
+import Prelude
+import Blockly.Types (Block, BlocklyState, Workspace)
+import Data.Array as Array
+import Data.Either (Either(..))
+import Data.Function.Uncurried (Fn1, Fn4, Fn5, Fn6, runFn1, runFn4, runFn5, runFn6)
+import Data.Maybe (Maybe)
+import Effect (Effect)
+import Effect.Uncurried (EffectFn1, EffectFn2, EffectFn4, runEffectFn1, runEffectFn2, runEffectFn4)
+import Partial.Unsafe (unsafePartial)
+
+type GeneratorFunction
+  = BlocklyState -> Block -> Either String String
+
+data Order
+  = Atomic
+  | None
+
+toNumber :: Order -> Number
+toNumber Atomic = 0.0
+
+toNumber None = 99.0
```

Comment:
Say what?

Reply:
This is some constant in the blockly API

Reply:
😲